pkg/local_object_storage/engine/put.go

```diff
@@ -163,6 +172,7 @@ func (e *StorageEngine) putToShard(sh hashedShard, ind int, pool util.WorkerPool
 
 		putSuccess = true
 	}); err != nil {
+		e.log.Warn("object put: pool task submitting", zap.Error(err))
```

Comment:
how this affects the system and/or what's the admin's reaction to this message?

Reply:
load to this shard currently is bigger than it can handle. also, we have a system that says where to put an object but now (this line) it is not gonna work: an object goes to another shard (we had (have?) bugs that relate such cases). in fact, it always bothered me that object put was always kinda random, you never know from logs if this placement is ok or a "bad" shard was taken as a best-effort

Reply:
this PR (and the issue) is a real example BTW. if i had this log, i wouldn't have had to run this test so many times trying to understand what was happening, it would have taken 1 min looking at logs